docs/walkthroughs/analysis-target-configuration.md

```diff
@@ -151,3 +151,32 @@ targets:
     - type: setuptools
       path: utils/helpers/
 ```
+
+## Target Filtering for Submodules
+
+For some package manager, you may have submodules or sub-projects within a single project
+that you are analyzing. And you may want to analyze only specifics sub project in some cases.
+
+Here is example with gradle:
```

Comment:
```suggestion
Here is an example with gradle:
```

Would it make this doc harder for users if we just link to the section in `gradle.md` that has this? That would avoid us having to maintain the same docs in two places.